index.html

```diff
@@ -200,6 +200,35 @@ <h3 class="iconic"></h3>Skills
                 </div>
             </div>
     </section>
+
+    <section class="reach">
+        <h1 class="int">I'm always interested in hearing about new projects, so if you'd like to chat please get in
+            touch.</h1>
```

Comment:
- [ ] KIndly make sure h1 is used only once on your portfolio page in order to respect the semantic HTML rules :+1: